source/index.ts

```diff
@@ -101,3 +107,33 @@ export function errorHandlerMiddleware(err: Error, req: any, resp: any, next: an
   backtraceClient.reportSync(err, { ...req, ...resp });
   next(err);
 }
+
+  /**
+   * Record an event in the breadcrumbs buffer
+   * @param message description of event
+   * @param attributes object of values related to event
+   * @param timestamp time of event
+   * @param level log level
+   * @param type type of event (e.g. "USER")
+   */
+  export function leaveBreadcrumb(
+    message: string,
+    attributes?: object,
+    timestamp?: number,
```

Comment:
Should we generate timestamp for user? I think it's not necessary to require it every time. What do you think?

Reply:
It is generated in the Breadcrumbs class method.